packages/mocks/pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.coverage.run]
+branch = true
+source = [ "mocks" ]
+
+[tool.coverage.report]
+exclude_lines = ['if TYPE_CHECKING:']
+show_missing = true
+
+[tool.pytest.ini_options]
+addopts = "--cov --cov-report html:'../../coverage/packages/mocks/html' --cov-report xml:'../../coverage/packages/mocks/coverage.xml' --html='../../reports/packages/mocks/unittests/html/index.html' --junitxml='../../reports/packages/mocks/unittests/junit.xml'"
```

Comment:
Remove "packages" from the options here: we want coverage to output to <repo root>/coverage/mocks not <repo root>/coverage/packages/mocks